src/Spago/Config.hs

```diff
@@ -2,6 +2,7 @@
 module Spago.Config
   ( makeConfig
   , ensureConfig
+  , parseConfig
```

Comment:
I'd like to not expose `parseConfig`, could we use `ensureConfig` instead since it takes care of some more things?

Reply:
Sure